config.py

```diff
@@ -25,6 +27,10 @@ class FilterRule(BaseModel):
 class Settings(BaseSettings):
     port_client_id: str
     port_client_secret: str
+    github_api_url: str = ""
```

Comment:
isn't github_api_url is always the same?

Reply:
I'll set a default, but I'm not sure